kuma/static/styles/base/elements/document.scss

```diff
@@ -8,6 +8,7 @@ html {
 }
 
 body {
-    line-height: 1.5;
+    line-height: 1.6;
     @include set-site-font-family();
+    @include set-font-size(15px);
```

Comment:
I don't normally like to reduce the font size below 16px, but Verdana is big. Playing with https://meowni.ca/font-style-matcher/ shows that 15px (with some tiny tweaks like the line-height and some letter-spacing) is nearly identical to 16px Open Sans.

Reply:
You can use `$body-font-size` here now.

Reply:
👍